department-of-veterans-affairs/caseflow
# Save time to the backend

Resolves #7620 #7491 

### Description
This PR adds a link to case details and saves time updates to the backend.

### Testing Plan
1. Go to the daily docket
1. Click on one of the veteran's vbms ids and confirm you get to case details
1. Update a time, confirm the time updates in the UI and the database
1. Repeat in other time zones
